web/src/features/charts/MissingExchangeData.tsx

```diff
@@ -0,0 +1,46 @@
+import { useTranslation } from 'react-i18next';
+
+import { useNetExchangeChartData } from './hooks/useNetExchangeChartData';
+import { AreaGraphElement } from './types';
+
+const SIGNIFICANT_THRESHOLD = 0.2;
+
+const getTotal = (object: object) => Object.keys(object).length;
+const getNulls = (object: object) =>
+  Object.values(object).filter((v) => v === null).length;
+
+function getIsMissingSignificantExchangeData(chartData: AreaGraphElement[]) {
+  let nullCount = 0;
+  let total = 0;
+  for (const exchange of chartData.map(({ meta: { exchange } }) => exchange)) {
+    nullCount += getNulls(exchange);
+    total += getTotal(exchange);
+  }
+  return nullCount / total >= SIGNIFICANT_THRESHOLD;
+}
+
+export function MissingExchangeDataDisclaimer() {
+  const { data, isLoading, isError } = useNetExchangeChartData();
+  const { t } = useTranslation();
+
+  if (isLoading || isError || !data) {
+    return null;
+  }
+
+  const { chartData } = data;
+
+  const date = new Date();
+  date.setDate(date.getDate() - 1);
+  date.setHours(0, 0, 0, 0);
```

Comment:
Ah JS you keep messing with me...

Why can I set seconds in the setHours function? You make no sense!

Reply:
🤣 😭

Reply:
This is so dumb!
```
parseInt(0.5); // -> 0
parseInt(0.05); // -> 0
parseInt(0.005); // -> 0
parseInt(0.0005); // -> 0
parseInt(0.00005); // -> 0
parseInt(0.000005); // -> 0
parseInt(0.0000005); // -> 5
```